share/picongpu/pypicongpu/template/include/picongpu/param/density.param.mustache

```diff
@@ -146,6 +146,39 @@ namespace picongpu
             using init_{{{placed_species_initial.typename}}} = FreeFormulaImpl<init_{{{placed_species_initial.typename}}}_functor>;
           {{/profile.typeID.foil}}
 
-        {{/species_initmanager.operations.simple_density}}
-    } // namespace densityProfiles::pypicongpu
-} // namespace picongpu
+           {{#profile.typeID.cylinder}}
+
+
+           struct init_{{{placed_species_initial.typename}}}_CylyndricalParam
+           {
+              static constexpr float3_64  centerPosition_SI{
+                  {{{profile.data.center_position_si.0.component}}},
+                  {{{profile.data.center_position_si.1.component}}},
+                  {{{profile.data.center_position_si.2.component}}}
+              };
+              static constexpr float3_X jetAxis{
+                  {{{profile.data.cylinder_axis.0.component}}}_X,
+                  {{{profile.data.cylinder_axis.1.component}}}_X,
+                  {{{profile.data.cylinder_axis.2.component}}}_X
```

Comment:
Just to mention it, there is a mustache syntax for list with variable length, see for example [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/7ebeee98def059f32968c1075617c7620aed19c3/share/picongpu/pypicongpu/template/include/picongpu/param/speciesDefinition.param.mustache#L67-L74)